src/components/barChart/barChart.js

```diff
@@ -0,0 +1,73 @@
+"use strict";
```

Comment:
you do not need use strict  -- if you use es5 modules -- use string is by default